api/test/wfl/integration/executor_test.clj

```diff
@@ -267,12 +310,83 @@
          (is (== 1 (stage/queue-length executor)))
          (is (not (stage/done? executor)))))))
 
+(deftest test-retry-terra-executor
```

Comment:
my goodness... I'll need some time for this!

Reply:
Some time and a :beer: :)

Reply:
(That being said, if it would help to walk through verbally what I'm testing LMK.)

Reply:
WOW, I need a ☕  before going through this …